daemon/integration/python/daemon_cli.py

```diff
@@ -0,0 +1,103 @@
+"""
+Usage:
+  daemon_cli.py createSnapshot <srcDir>
```

Comment:
not going to comment on the interface of this because you said it wasn't the right time to review it now. : )